src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -13,4 +13,7 @@
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_REMARK = new Prefix("r/");
     public static final Prefix PREFIX_FLAG = new Prefix("-");
+
+    public static final Prefix PREFIX_ALIAS_COMMAND = new Prefix("c/");
```

Comment:
Should be a different prefix to `category`.

Reply:
Understood. I updated the prefix for alias to be "cmd/" for command and "al/" for the alias name